<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenBKZ</class>
 <widget class="QMainWindow" name="OpenBKZ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>601</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>7800</width>
    <height>6100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OpenBKZ</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="7">
     <widget class="QPushButton" name="thumbPage">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Back to Page</string>
      </property>
     </widget>
    </item>
    <item row="24" column="3">
     <widget class="QComboBox" name="search_type">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Pages</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Chapters</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="24" column="5">
     <widget class="QPushButton" name="prevButton">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Prev</string>
      </property>
     </widget>
    </item>
    <item row="24" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>OpenBKZ Alpha v0.4</string>
      </property>
     </widget>
    </item>
    <item row="24" column="7">
     <widget class="QComboBox" name="user_rating">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>☆☆☆☆☆</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>★☆☆☆☆</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>★★☆☆☆</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>★★★☆☆</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>★★★★☆</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>★★★★★</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="24" column="4">
     <widget class="QLineEdit" name="lineEdit_page">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>search</string>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="24" column="2">
     <widget class="QPushButton" name="enable_stats_button">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Disable Statistics</string>
      </property>
     </widget>
    </item>
    <item row="24" column="6">
     <widget class="QPushButton" name="nextButton">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>next</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="11" colspan="6">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>630</width>
        <height>475</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>6300</width>
        <height>6300</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::RubberBandDrag</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QToolButton" name="toolButton">
      <property name="maximumSize">
       <size>
        <width>105</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Choose Book</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QPushButton" name="saveBookButton">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Bookmark</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QPushButton" name="pushNoteButton">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Add Note</string>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QPushButton" name="viewStats">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Statistics</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QSlider" name="fontSizeSlider">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="minimum">
       <number>4</number>
      </property>
      <property name="maximum">
       <number>64</number>
      </property>
      <property name="sliderPosition">
       <number>13</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
